src/tools/refrescoTokenZoom.js

```diff
@@ -0,0 +1,42 @@
+const got = require('got')
+
+const ZoomDatosUsuarios = require('../models/zoomDatosUsuarios')
+
+const ZOOM_CLIENT_ID = "BqYXOyymQ_OGhBXQKV653A";
+const ZOOM_CLIENT_SECRET = "jvchOnsZAjVtHRAaYP7xEj95XaiwX6el";
+
+function refrescoTokenZoom() {
+    (async () => {
+        try {
+
+            const usuarios = await ZoomDatosUsuarios.find({}).exec()
+            
+            for (const usuario of usuarios) {
+                if(usuario.refresh_token != null) {
+
+                    const respuesta = await got.post('https://zoom.us/oauth/token'
+                        + '?grant_type=refresh_token'
+                        + '&refresh_token=' + usuario.refresh_token, {
+                            headers: {
+                                "Authorization": "Basic " + Buffer.from(ZOOM_CLIENT_ID + ':' + ZOOM_CLIENT_SECRET).toString('base64')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7157741)
```suggestion
                                "Authorization": "Basic " + Buffer.from(ZOOM_CLIENT_ID + ":" + ZOOM_CLIENT_SECRET).toString('base64')
```